docs/articles/nunit/writing-tests/attributes/cancelafter.md

```diff
@@ -0,0 +1,52 @@
+# CancelAfter
+
+Normally, NUnit simply runs tests and waits for them to terminate—the test is allowed to run indefinitely. For certain kinds of tests, however, it may be desired to specify a timeout value.
```

Comment:
```suggestion
Normally, NUnit simply runs tests and waits for them to terminate -- the test is allowed to run indefinitely. For certain kinds of tests, however, it may be desirable to specify a timeout value.
```
Minor wording

Reply:
@SeanKilleen This was literally copied from `timeout.md`. I have no problems with changing this, but then we should also change the other.